template/content/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Bit of an aside, but if we're changing it in a scaffold shouldn't we consider changing it in the standard?